packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -111,13 +119,11 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
   }
 
   internal fun setTintColor(color: Int?) {
-    if (color == null) {
-      clearColorFilter()
-    } else {
-      setColorFilter(color, PorterDuff.Mode.SRC_IN)
-    }
+    color?.let { setColorFilter(it, PorterDuff.Mode.SRC_IN) } ?: clearColorFilter()
```

Comment:
This maintains the same behavior, if it's less readable than the original, I'll revert this